homeassistant/components/logger.py

```diff
@@ -23,6 +27,22 @@
 LOGGER_DEFAULT = 'default'
 LOGGER_LOGS = 'logs'
 
+_LOGS_SCHEMA = vol.All(
+    cv.ensure_list,
+    [
+        vol.Schema({
+            cv.string: vol.In(vol.Lower(list(LOGSEVERITY))),
```

Comment:
What about making this `cv.slug:` iso `cv.string:`? It forces lowercase and allowable characters